Linkbus/Linkbus/RouteController.swift

```diff
@@ -29,7 +29,7 @@ extension RouteController {
         
         dispatchGroup.enter()
         fetchCsbsjuApi { apiResponse in
-            if let success = apiResponse {
```

Comment:
Let's keep this in for now so that we can use success for network error catching